how-to/workspace-platform-starter/client/src/framework/shapes/platform-shapes.ts

```diff
@@ -50,186 +50,155 @@ export interface PlatformStorageMetadata {
 }
 
 /**
- * A request type for the WorkspaceEndpoint that gets all saved workspace entries
+ * A request type for the Storage Endpoint that gets a specific entry
  */
-export interface EndpointWorkspaceListRequest {
+export interface EndpointGetRequest {
 	/**
 	 * The id of the platform making the request
 	 */
 	platform: string;
 	/**
-	 * Optional query that limits the workspaces.
+	 * The id of the entry to get.
 	 */
-	query?: string;
+	id: string;
```

Comment:
Is this a breaking change? We now have others using this so should only do breaking changes for specific reasons.

Reply:
Nope, just the way the diff displays